chapter_5_practic/task12.cpp

```diff
@@ -0,0 +1,100 @@
+// Revise the four - function fraction calculator from Exercise 12, Chapter 4, so that it uses
+// functions for each of the four arithmetic operations.They can be called fadd(), fsub(),
+// fmul(), and fdiv().Each of these functions should take two arguments of type struct
+// fraction, and return an argument of the same type.
+
+#include <iostream>
+
+struct fraction
+{
+	int nominator = 0;
```

Comment:
`numerator`

Reply:
https://github.com/Dmitriy-Romanchuk/master/pull/5/commits/381b0809f897ecb9e1ba28e1619999d4b570e78f
But I'm not sure if I did it right